docs/pipeline-components-and-applications/snowplow-mini/setup-guide-for-aws/index.md

```diff
@@ -18,16 +18,17 @@ We offer Snowplow Mini in 3 different sizes. To decide on which size of Snowplow
 
 ### large & xlarge & xxlarge
 
-Until `0.6.0`, Snowplow Mini was being used inside AWS `t2.medium`, `n1-standard-1` in GCP, instances and it served well for demonstration purposes. However, we observed that Snowplow Mini started exceeding its initial motivation and machine resources started to become an obstacle, causing issues with Elasticsearch etc. This is why, Mini is available at 3 different sizes.
+Until `0.6.0`, Snowplow Mini was being used inside AWS `t2.medium`, `n1-standard-1` in GCP, instances and it served well for demonstration purposes. However, we observed that Snowplow Mini started exceeding its initial motivation and machine resources started to become an obstacle, causing issues with Opensearch etc. This is why, Mini is available at 3 different sizes.
```

Comment:
I think by now we can just say “Mini is available in 3 different sizes” and cut the rest of this paragraph